### PR TITLE
feat: labs

### PR DESCRIPTION
**Pull request by yanzhihong23**
closes #

image

### CODE REVIEW

File: packages/dashboard/src/pages/Labs/constants.ts [hunk 1]
```diff
@@ -13,3 +13,54 @@ export const PLUGIN_IDS = {
     POLL: 'com.maskbook.poll',
     PETS: 'com.maskbook.pets',
 }
+
+type TUTORIAL_URLS_OPT = {
+    [key: string]: string
+}
+
+export const TUTORIAL_URLS_EN: TUTORIAL_URLS_OPT = {
```
Comment (guanbinrui) [inline packages/dashboard/src/pages/Labs/constants.ts:21]: Suppose you need this configuration to update on the fly. Would you please take a look at https://github.com/DimensionDev/Maskbook-Configuration?

**Review by guanbinrui — DISMISSED** [review]

File: packages/mask/.webpack/config.ts [hunk 2]
```diff
@@ -111,6 +111,7 @@ export function createConfiguration(rawFlags: BuildFlags): Configuration {
                 },
             },
             rules: [
+                { test: /\.svg$/, use: ['@svgr/webpack'] },
```
Comment (Jack-Works) [inline packages/mask/.webpack/config.ts:114]: Please do not add this.

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/dashboard/src/pages/Labs/index.tsx [hunk 3]
```diff
@@ -83,25 +194,23 @@ export default function Plugins() {
...
 
-    function onExplore(id: string) {
-        if (id === PLUGIN_IDS.TRANSAK) {
-            openTransakDialog()
-        } else if (id === PLUGIN_IDS.SWAP) {
-            openSwapDialog()
+    function onTutorial(id: string) {
+        const url = TUTORIAL_URLS_EN[id]
+        if (url) {
+            window.open(url)
```
Comment (guanbinrui) [inline packages/dashboard/src/pages/Labs/index.tsx:206]:
```suggestion
            window.open(url, '_blank', 'noopener noreferrer')
```

**Review by Jack-Works — APPROVED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Comment by wenluomask** [comment]
1. Change Maskbox‘s link to https://realmasknetwork.notion.site/How-to-participate-in-a-MaskBox-sale-d0941687649a4ef7a38d71f23ecbe4da 
2. dHEDGE's link change to https://realmasknetwork.notion.site/Invest-in-your-favourite-fund-manager-via-dHEDGE-on-Twitter-ETH-and-Polygon-fb00ff2e626949279c83b59ed9207b9a
3. Add a full stop in Maskbox;
Screen Shot 2021-12-02 at 3 17 16 PM

**Comment by yanzhihong23** [comment]
> 

updated

**Comment by wenluomask** [comment]
Checked.

**Review by guanbinrui — APPROVED** [review]

**Review by NikoManiac — APPROVED** [review]

**Pull request merged**